.travis.yml

```diff
@@ -12,7 +11,7 @@ cache:
 
 env:
   - EMBER_TRY_SCENARIO=default
-  - EMBER_TRY_SCENARIO=ember-1-13
+  - EMBER_TRY_SCENARIO=ember-1.13
```

Comment:
I'll probably drop support for `1.13`, as it will make moving to Glimmer 2 features slower later on.